### ISSUE TITLE
ModuleNotFoundError: No module named 'blessed'

### ISSUE THREAD

**Issue by anthonydb** [dalibo/pg_activity#196]
I'm on macOS Catalina and receive the above error when attempting to launch pg_activity. Funny thing is that I do have blessed installed in my env -- version 1.18.0.

I'm running pg_activity version  2.1.2.

Full traceback:

```
Traceback (most recent call last):
  File "/Users/******/.pyenv/versions/data-3.9.1/bin/pg_activity", line 35, in <module>
    from blessed import Terminal
ModuleNotFoundError: No module named 'blessed'
```

**Comment by dlax** [comment]
Can you try to install as `pip install https://github.com/dalibo/pg_activity/archive/entry-points.zip`? This corresponds to #198; if it works for you, I'll merge it and ship a release. Thanks.

**Comment by anthonydb** [comment]
Unfortunately, that did not fix my issue:

```
pip install https://github.com/dalibo/pg_activity/archive/entry-points.zip
Collecting https://github.com/dalibo/pg_activity/archive/entry-points.zip
  Downloading https://github.com/dalibo/pg_activity/archive/entry-points.zip
     \ 296 kB 1.0 MB/s
  Installing build dependencies ... done
  Getting requirements to build wheel ... done
    Preparing wheel metadata ... done
Requirement already satisfied: attrs in ./.pyenv/versions/3.9.1/envs/data-3.9.1/lib/python3.9/site-packages (from pg-activity==2.1.2) (20.3.0)
Requirement already satisfied: humanize in ./.pyenv/versions/3.9.1/envs/data-3.9.1/lib/python3.9/site-packages (from pg-activity==2.1.2) (3.2.0)
Requirement already satisfied: psutil>=2.0.0 in ./.pyenv/versions/3.9.1/envs/data-3.9.1/lib/python3.9/site-packages (from pg-activity==2.1.2) (5.8.0)
Requirement already satisfied: blessed in ./.pyenv/versions/3.9.1/envs/data-3.9.1/lib/python3.9/site-packages (from pg-activity==2.1.2) (1.18.0)
Requirement already satisfied: six>=1.9.0 in ./.pyenv/versions/3.9.1/envs/data-3.9.1/lib/python3.9/site-packages (from blessed->pg-activity==2.1.2) (1.15.0)
Requirement already satisfied: wcwidth>=0.1.4 in ./.pyenv/versions/3.9.1/envs/data-3.9.1/lib/python3.9/site-packages (from blessed->pg-activity==2.1.2) (0.2.5)
Requirement already satisfied: setuptools in ./.pyenv/versions/3.9.1/envs/data-3.9.1/lib/python3.9/site-packages (from humanize->pg-activity==2.1.2) (54.1.2)
(data-3.9.1)[debarrosa ~]$ pg_activity -h wsj-data-journalism-db-1.chthw1jg4rom.us-east-1.rds.amazonaws.com -d us_fed_contracts -U wsj_scraper --rds
Traceback (most recent call last):
  File "/Users/debarrosa/.pyenv/versions/data-3.9.1/bin/pg_activity", line 35, in <module>
    from blessed import Terminal
ModuleNotFoundError: No module named 'blessed'
```

**Comment by dlax** [comment]
> ```
>   File "/Users/******/.pyenv/versions/data-3.9.1/bin/pg_activity", line 35, in <module>
>     from blessed import Terminal
> ModuleNotFoundError: No module named 'blessed'
> ```

That file should not be that long (35 lines here, only 10 lines in a normal installation); it seems that the installation did nothing, probably with a "Requirement already satisfied" message.

Can you try `pip install -I https://github.com/dalibo/pg_activity/archive/entry-points.zip`?

**Comment by anthonydb** [comment]
That worked and fixed the issue!

**Comment by dlax** [comment]
Just released version 2.1.3 with the fix; you can now `pip install -U pg_activity`.
